<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>personForm</class>
 <widget class="QWidget" name="personForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_Class">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>390</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_line2">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>391</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_person_mess">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_line3">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>390</y>
     <width>291</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_myquit">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>430</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_line4">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>430</y>
     <width>291</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_class">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_person_message">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_quit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_class2">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>405</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>等级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_person_message2">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>13</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>个人信息设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_quit2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>443</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_service3">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>403</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_class3">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>405</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文仿宋</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>暂无</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_L">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>13</y>
     <width>16</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_L6">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>403</y>
     <width>16</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="reviseBtn">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>340</y>
     <width>41</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UserName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PhoneNum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>135</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Password">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>196</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>73</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneNumEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>133</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>193</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmailEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>253</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Email">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>253</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
